sdcm/cluster_k8s/mini_k8s.py

```diff
@@ -234,6 +234,7 @@ def local_kubectl_version(self):  # pylint: disable=no-self-use
         #   Client Version: v1.18.5
         return LOCALRUNNER.run("kubectl version --client --short").stdout.rsplit(None, 1)[-1][1:]
 
+    @retrying(n=3, sleep_time=10, message="Pulling docker image")
```

Comment:
how this is related to the PR ? do you call it here ?

Reply:
It's not related, I had issues with it and wanted to fix it. I'll move it to different commit/PR